validator/sawtooth_validator/networking/interconnect.py

```diff
@@ -345,9 +345,14 @@ def _receive_message(self):
                 if self._socket.getsockopt(zmq.TYPE) == zmq.ROUTER:
                     zmq_identity, msg_bytes = \
                         yield from self._socket.recv_multipart()
-                    self._received_from_identity(zmq_identity)
-                    self._dispatcher_queue.put_nowait(
-                        (zmq_identity, msg_bytes))
+                    if msg_bytes == b'':
+			# send ACK for connection probes
+                        LOGGER.debug("ROUTER PROBE FROM %s",zmq_identity)
```

Comment:
How often will these logging messages be generated?

Reply:
Every probe request, i.e., once per TP connection.